src/Symfony/Component/Workflow/TraceableWorkflow.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Workflow;
```

Comment:
```suggestion
namespace Symfony\Component\Workflow\Debug;
```

i am used to seeing some debug classes inside a debug subnamespace